stream-legal-entity/legal-entity-http/src/main/java/com/backbase/stream/LegalEntityHttpApplication.java

```diff
@@ -28,4 +31,15 @@ public SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http)
             .build();
     }
 
+    /**
+     * To support tracing requests to the services.
+     *
+     * @return In memory HttpTraceRepository.
+     */
+    @Bean
+    @ConditionalOnExpression("${management.endpoints.enabled-by-default:false} or ${management.trace.http.enabled:false}")
```

Comment:
why is the repository created only when this conditions are false?

Reply:
and where is this used?

Reply:
@JuanAnsio  this is auxiliary to the actuator/httptrace endpoint, you can find more information here: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.2-Release-Notes#actuator-http-trace-and-auditing-are-disabled-by-default